Feature/post devops best practices

cristofer-dev opened the pull request:
### Draft of the post suggested good practices

Include
- Suggested version
- Naming branches
- Tags
- Commit messages
- Merging type suggested

cristofer-dev committed:
ADD draft post

 Changes to be committed:
	new file:   src/posts/2019-01-22-devops.md

cristofer-dev committed:
DRAFT content

 Changes to be committed:
	modified:   src/posts/2019-01-22-devops.md

cristofer-dev committed:
DRAFT Rules for commit message

cristofer-dev committed:
UPDATE titles sections

 Changes to be committed:
	modified:   src/posts/2019-01-22-devops.md

Xabadu commented on src/posts/2019-01-22-devops.md:
```diff
@@ -0,0 +1,117 @@
+image: aws.svg
+---
+
+¿Que número de version sigue? ¿el tag va en mi rama? ¿bugfix/ o hotfix?
```
```suggestion
¿Que número de versión sigue? ¿el tag va en mi rama? ¿bugfix o hotfix?
```

Xabadu commented on src/posts/2019-01-22-devops.md:
```diff
@@ -0,0 +1,117 @@
+y : Agregamos una nueva característica
+z : Solucionamos un bug
+
+v0.1.0: Versión de desarrollo inicial con sus primeras característica funcionales.
```
```suggestion
v0.1.0: Versión de desarrollo inicial con sus primeras características funcionales.
```

Xabadu commented on src/posts/2019-01-22-devops.md:
```diff
@@ -0,0 +1,117 @@
+z : Solucionamos un bug
+
+v0.1.0: Versión de desarrollo inicial con sus primeras característica funcionales.
+v0.2.0: Versión de desarrollo inicial con nuevas característica funcionales.
```
```suggestion
v0.2.0: Versión de desarrollo inicial con nuevas características funcionales.
```

Xabadu commented on src/posts/2019-01-22-devops.md:
```diff
@@ -0,0 +1,117 @@
+v0.2.1: Versión de desarrollo inicial que soluciona un bug.
+v1.0.0: Versión 1 estable.
+v1.0.1: Soluciona bugs de la versión 1.
+v1.2.0: Agrega nueva característica a la version 1.
```
```suggestion
v1.2.0: Agrega nueva características a la versión 1.
```

Xabadu commented on src/posts/2019-01-22-devops.md:
```diff
@@ -0,0 +1,117 @@
+
+### Usando tags
+
+- Importante usar tags para llevar el control de nuestros release.
```
```suggestion
- Importante usar tags para llevar el control de nuestros releases.
```

Xabadu commented on src/posts/2019-01-22-devops.md:
```diff
@@ -0,0 +1,117 @@
+### Usando tags
+
+- Importante usar tags para llevar el control de nuestros release.
+- Usar semver
```
```suggestion
- Usar semver.
```

Xabadu commented on src/posts/2019-01-22-devops.md:
```diff
@@ -0,0 +1,117 @@
+
+- Importante usar tags para llevar el control de nuestros release.
+- Usar semver
+- Tags en master (despúes del merge)
```
```suggestion
- Tags en master (despúes del merge).
```

Xabadu commented on src/posts/2019-01-22-devops.md:
```diff
@@ -0,0 +1,117 @@
+Las siete reglas de un gran mensaje de Git. [ref](https://chris.beams.io/posts/git-commit/)
+
+- Separar el asunto del cuerpo con una línea en blanco.
+- Limite la línea de asunto a 50 caracteres
```
```suggestion
- Limite la línea de asunto a 50 caracteres.
```

Xabadu commented on src/posts/2019-01-22-devops.md:
```diff
@@ -0,0 +1,117 @@
+
+- Separar el asunto del cuerpo con una línea en blanco.
+- Limite la línea de asunto a 50 caracteres
+- Capitalizar la línea de asunto
```
```suggestion
- Capitalizar la línea de asunto.
```

Xabadu commented on src/posts/2019-01-22-devops.md:
```diff
@@ -0,0 +1,117 @@
+- Separar el asunto del cuerpo con una línea en blanco.
+- Limite la línea de asunto a 50 caracteres
+- Capitalizar la línea de asunto
+- **No** termine la línea de asunto con un punto
```
```suggestion
- **No** termine la línea de asunto con un punto.
```

Xabadu commented on src/posts/2019-01-22-devops.md:
```diff
@@ -0,0 +1,117 @@
+- Limite la línea de asunto a 50 caracteres
+- Capitalizar la línea de asunto
+- **No** termine la línea de asunto con un punto
+- Utilice el modo imperativo en la línea de asunto
```
```suggestion
- Utilice el modo imperativo en la línea de asunto.
```

Xabadu commented on src/posts/2019-01-22-devops.md:
```diff
@@ -0,0 +1,117 @@
+- Capitalizar la línea de asunto
+- **No** termine la línea de asunto con un punto
+- Utilice el modo imperativo en la línea de asunto
+- Cada linea del cuerpo no debe exceder los 72 caracteres por linea.
```
```suggestion
- Cada línea del cuerpo no debe exceder los 72 caracteres por línea.
```

Xabadu commented on src/posts/2019-01-22-devops.md:
```diff
@@ -0,0 +1,117 @@
+- **No** termine la línea de asunto con un punto
+- Utilice el modo imperativo en la línea de asunto
+- Cada linea del cuerpo no debe exceder los 72 caracteres por linea.
+- Usa el cuerpo para explicar **qué** y **por qué** en lugar del **cómo**
```
```suggestion
- Usa el cuerpo para explicar **qué** y **por qué** en lugar del **cómo**.
```

Xabadu commented on src/posts/2019-01-22-devops.md:
```diff
@@ -0,0 +1,117 @@
+Resume los cambios en alrededor de 50 caracteres o menos
+
+Texto explicativo más detallado, si es necesario. Envuélvelo a unos 72
+personajes más o menos. En algunos contextos, la primera línea se trata
```
```suggestion
caracteres más o menos. En algunos contextos, la primera línea se trata
```

Xabadu commented on src/posts/2019-01-22-devops.md:
```diff
@@ -0,0 +1,117 @@
+
+Texto explicativo más detallado, si es necesario. Envuélvelo a unos 72
+personajes más o menos. En algunos contextos, la primera línea se trata
+como la Asunto de la confirmación y el resto del texto como cuerpo. la
```
```suggestion
como el asunto y el resto del texto como cuerpo. La
```

Xabadu commented on src/posts/2019-01-22-devops.md:
```diff
@@ -0,0 +1,117 @@
+personajes más o menos. En algunos contextos, la primera línea se trata
+como la Asunto de la confirmación y el resto del texto como cuerpo. la
+línea en blanco que separa el resumen del cuerpo es crítica (a menos
+que omites el cuerpo por completo); varias herramientas como `log`,
```
```suggestion
que omitas el cuerpo por completo); Varias herramientas como `log`,
```

Xabadu commented on src/posts/2019-01-22-devops.md:
```diff
@@ -0,0 +1,117 @@
+como la Asunto de la confirmación y el resto del texto como cuerpo. la
+línea en blanco que separa el resumen del cuerpo es crítica (a menos
+que omites el cuerpo por completo); varias herramientas como `log`,
+`shortlog` y `rebase` puede confundirse si no hay linea en blanco.
```
```suggestion
`shortlog` y `rebase` pueden confundirse si no hay línea en blanco.
```

Xabadu commented on src/posts/2019-01-22-devops.md:
```diff
@@ -0,0 +1,117 @@
+`shortlog` y `rebase` puede confundirse si no hay linea en blanco.
+
+En el cuerpo explica el problema que este commit está resolviendo.
+Centrarse en por qué usted están haciendo este cambio en lugar de cómo
```
```suggestion
Centrarse en por qué usted está haciendo este cambio en lugar de cómo
```

Xabadu commented on src/posts/2019-01-22-devops.md:
```diff
@@ -0,0 +1,117 @@
+
+ - Normalmente se usa un guión o un asterisco para la lista, precedido
+   por un solo espacio, con líneas en blanco en el medio, pero las convenciones
+   variar aqui
```
```suggestion
   variar aquí.
```

Xabadu reviewed (changes requested):
Algunas correcciones específicas de ortografía, pero adicionalmente, hay algunos temas de redacción que le daría una vuelta adicional, algunas partes del texto están redactados en primera persona (cambiando de tù a ud./uds. y viceversa) :)

Pero en general 👌

Xabadu committed:
Update src/posts/2019-01-22-devops.md

Co-Authored-By: cristofer-dev <cristofer-dev@users.noreply.github.com>

Xabadu committed:
Update src/posts/2019-01-22-devops.md

Co-Authored-By: cristofer-dev <cristofer-dev@users.noreply.github.com>

Xabadu committed:
Update src/posts/2019-01-22-devops.md

Co-Authored-By: cristofer-dev <cristofer-dev@users.noreply.github.com>

Xabadu committed:
Update src/posts/2019-01-22-devops.md

Co-Authored-By: cristofer-dev <cristofer-dev@users.noreply.github.com>

Xabadu committed:
Apply suggestions from code review

Co-Authored-By: cristofer-dev <cristofer-dev@users.noreply.github.com>

cristofer-dev committed:
UPDATE post good practices

 Changes to be committed:
	modified:   src/assets/img/dev-tools.svg
	modified:   src/posts/2019-01-22-devops.md

cristofer-dev committed:
UPDATE footer in post

 Changes to be committed:
	modified:   src/posts/2018-08-24-tools.md
	modified:   src/posts/2019-01-22-devops.md

cristofer-dev committed:
FIX minors post good practices

 Changes to be committed:
	modified:   src/posts/2019-01-22-devops.md
